src/openrct2-ui/windows/Window.h

```diff
@@ -204,6 +204,8 @@ WindowBase* WindowSceneryScatterOpen();
 WindowBase* WindowPatrolAreaOpen(EntityId staffId);
 EntityId WindowPatrolAreaGetCurrentStaffId();
 
+WindowBase* ResetShortcutKeysPromptOpen();
```

Comment:
No need to expose this to window.h

Reply:
What's the alternative? At this time, all windows opening another use a definition from Window.h. For example, WindowRideDemolishPromptOpen.

Reply:
WindowRideDemolishPromptOpen is defined in a standalone file so it needs to be in Window.h so that the ride window and all the other callers can call it but we are defining `ResetShortcutKeysPromptOpen` in ShortcutKeys.cpp and the Shortcut window is its only caller.

Reply:
Check, I moved the definition.